kuka_kr210_support/launch/test_kr210l150-2.launch

```diff
@@ -0,0 +1,8 @@
+<launch>
+  <include file="$(find kuka_kr210_support)/launch/load_kr210l150-2.launch" />
```

Comment:
Same as above... update to use new variant arguments

Reply:
This file is non-existent now.